### PR TITLE
334 add new settings

### LINKED ISSUE #334: Add new features to Ng for v4.15
**Is your feature request related to a problem? Please describe.**

Go through change log for 4.15 in ids enterprise https://github.com/infor-design/enterprise/blob/4.15.x/docs/CHANGELOG.md

Add any new settings or make issues for the more complicated things that cannot be hit on first pass (which will be listed here)

- [x] #1422 - Add setting for filtered child rows to datagrid settings as per https://github.com/infor-design/enterprise/pull/1480/files#diff-7a47cbfa760f2056d4f9b91cf583d3fcR109
- [x] #653 - Add 'lookup' to the list of supported 'filterTypes' in the datagrid column option as per https://github.com/infor-design/enterprise/pull/1455/files#diff-7a47cbfa760f2056d4f9b91cf583d3fcR1220
- [x] #1303 - Add new methods to the datagrid (rowNode , cellNode  clearDirtyClass, clearDirty, clearDirtyRow, clearDirtyCell, dirtyCells) as per https://github.com/infor-design/enterprise/pull/1409/files#dff-7a47cbfa760f2056d4f9b91cf583d3fcR4623

### PR DESCRIPTION
**Pull request by pwpatton**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Updating to match EP apis

**Related github/jira issue (required)**:
Closes #334

### CODE REVIEW

File: projects/ids-enterprise-ng/src/lib/accordion/soho-accordion.component.ts [hunk 1]
```diff
@@ -201,6 +201,21 @@ export class SohoAccordionComponent implements AfterViewInit, AfterViewChecked,
     return this.options.alternate;
   }
 
+  /**
+   * Set the color scheme to alternate
```
Comment (fitzorama) [inline projects/ids-enterprise-ng/src/lib/accordion/soho-accordion.component.ts]: wrong comment

File: projects/ids-enterprise-ng/src/lib/datagrid/soho-datagrid.component.ts [hunk 2]
```diff
@@ -1833,6 +1864,31 @@ export class SohoDataGridComponent implements OnInit, AfterViewInit, OnDestroy,
...
+  }
+
+  /**
+   * Returns the cell dom node.
+   * @param  row The row index.
+   * @param  cell The cell index.
+   * @param  includeGroups If true groups are taken into account.
+   * @return The dom node
+   */
+  cellNode(row: number, cell: number, includeGroups: boolean): any {
```
[the remark this answers is not in the document]
Comment (pwpatton) [reply projects/ids-enterprise-ng/src/lib/datagrid/soho-datagrid.component.ts]: going to fix that in next release 5.2.x

File: projects/ids-enterprise-ng/src/lib/datagrid/soho-datagrid.component.ts [hunk 3]
```diff
@@ -1833,6 +1864,31 @@ export class SohoDataGridComponent implements OnInit, AfterViewInit, OnDestroy,
     });
   }
 
+  /**
+   * Returns the row dom jQuery node.
+   * @param  row The row index.
+   * @param  includeGroups If true groups are taken into account.
+   * @return The dom jQuery node
+   */
+  rowNode(row: number, includeGroups: boolean): any {
```
[the remark this answers is not in the document]
Comment (pwpatton) [reply projects/ids-enterprise-ng/src/lib/datagrid/soho-datagrid.component.ts]: going to fix that in next release 5.2.x

File: projects/ids-enterprise-ng/src/lib/field-filter/soho-field-filter.d.ts [hunk 4]
```diff
@@ -21,6 +21,19 @@ interface SohoFieldFilterStatic {
...
+   * returns {object} The current filter type
+   */
+  getFilterType(): any;
+
+  /**
+   * Set filter type to given value
+   * param {number|string} value to be set, index or value.
+   * returns {void}
+   */
+  setFilterType(value: any);
```
[the remark this answers is not in the document]
Comment (pwpatton) [reply projects/ids-enterprise-ng/src/lib/field-filter/soho-field-filter.d.ts]: Not sure about that. The docs for EP didn't mention any

File: projects/ids-enterprise-ng/src/lib/accordion/soho-accordion.component.ts (hunk 1 above)
Comment (pwpatton) [reply projects/ids-enterprise-ng/src/lib/accordion/soho-accordion.component.ts]: Fixed

**Pull request merged**